src/client/camera.cpp

```diff
@@ -281,6 +282,20 @@ void Camera::addArmInertia(f32 player_yaw)
 	}
 }
 
+void Camera::updateOffset()
+{
+	v3f cp = m_camera_position / BS;
+
+	// Update offset if too far away from the center of the map
+	m_camera_offset = v3s16(
+		floorf(cp.X / CAMERA_OFFSET_STEP) * CAMERA_OFFSET_STEP,
```

Comment:
@sfan5 This will also switch the offset at a fixed position.
Before the change it would not change the offset again, unless you actually move 200 nodes.

Reply:
I see. I missed that on the old code because it wasn't obvious.

No urgent need to change it back however IMO. We should be handling offset changes correctly in the first place, then the timing doesn't matter.